### PR TITLE
[TwitterSentiment] doc updates

### PR DESCRIPTION
**Pull request by ghelmling**

### CODE REVIEW

File: TwitterSentiment/README.rst [hunk 1]
```diff
@@ -1,60 +1,129 @@
...
+.. _OAuth access tokens: https://dev.twitter.com/oauth/overview/application-owner-access-tokens
 
+These configurations must be provided as runtime arguments to the flow prior to starting it, in
+order to use the ``TweetCollector`` flowlet. To avoid this, configure the ``disable.public``
+argument as described below.
+
+Flow Runtime Arguments
+----------------------
+When starting the ``SentimentAnalysisFlow`` flow from the UI, runtime arguments can be
+specified to enable tweet collection.  To add runtime arguments, click on the gear icon above
```
Comment (awholegunch) [inline TwitterSentiment/README.rst]: To add runtime arguments, click on the gear icon shown in the upper-right of the flow display.

File: TwitterSentiment/README.rst [hunk 2]
```diff
@@ -1,60 +1,129 @@
...
+shown above and to the right of the flow display.
+
+The following arguments are supported:
```
Comment (awholegunch) [inline TwitterSentiment/README.rst]: These arguments are supported:

File: TwitterSentiment/README.rst [hunk 3]
```diff
@@ -1,60 +1,129 @@
...
+=============
 
 Twitter Configuration
-=====================
-In order to utilize the TweetCollector flowlet, which pulls a small sample stream via the Twitter API, the API key and Access token must be configured.
-Follow the steps at the following page to obtain these credentials: [Twitter oauth access tokens](https://dev.twitter.com/oauth/overview/application-owner-access-tokens).
-These configurations must be provided as runtime arguments to the flow prior to starting it, in order to use the TweetCollector flowlet. To avoid this, see the "disable.public" argument option below.
+---------------------
+In order to utilize the ``TweetCollector`` flowlet, which pulls a small sample stream via the Twitter
+API, the Twitter API key and Access token to use must be configured. Follow the steps provided by
```
Comment (awholegunch) [inline TwitterSentiment/README.rst]: In order to utilize the `TweetCollector` flowlet, which pulls a small sample stream via the Twitter API, a Twitter API key and Access token must be obtained and configured. Follow the steps provided by Twitter to obtain `OAuth access tokens`_.

File: TwitterSentiment/README.rst [hunk 4]
```diff
@@ -1,60 +1,129 @@
...
+Run Web UI::
+
+  mvn -Pweb jetty:run [-Dcdap.host=hostname] [-Dcdap.port=port]
+
+(optionally use ``-Dcdap.host=hostname`` and ``-Dcdap.port=port`` to point to CDAP,
+``localhost:10000`` is used by default)
+
+Once the Web UI is running, it can be viewed at http://localhost:8080/TwitterSentiment/
+
+Configuration
```
Comment (abaranau) [inline TwitterSentiment/README.rst]: LEt's call it "processing realtime twitter data"..

**Comment by ghelmling** [comment]
Updated for review comments

**Pull request merged**